### PR TITLE
Update to cave lighting and Chad pos.

### PR DESCRIPTION
**Pull request by calebkrauter**

### CODE REVIEW

**Review by NathanHinthorne — APPROVED** [review]
Looks good!

File: js/classes/entities/midground/npc/village/blacksmith copy.js
Comment (b1gd3vd0g) [inline js/classes/entities/midground/npc/village/blacksmith copy.js:1]: what´s this for? Why a copy?

File: sprites/Sprite-0002.aseprite
Comment (b1gd3vd0g) [inline sprites/Sprite-0002.aseprite:1]: why is there an aseprite file saved here?

File: js/classes/entities/midground/npc/village/blacksmith copy.js
Comment (tclaar) [inline js/classes/entities/midground/npc/village/blacksmith copy.js:1]: What's this?

**Review by tclaar — APPROVED** [review]

**Review by b1gd3vd0g — APPROVED** [review]

**Pull request merged**